frontend/public/components/_dropdown.scss

```diff
@@ -193,15 +181,17 @@ $dropdown-link-hover-border: 1px solid #bee1f4;
   }
 }
 
-.co-namespace-selector__menu.dropdown-menu {
+.co-namespace-selector__menu.pf-c-dropdown__menu {
+  margin: 0;
   max-height: 60vh;
   max-width: 100%;
   min-width: 210px;
   overflow-y: auto;
+  top: 41px; // namespace selector bar height - 1px (bottom border)
```

Comment:
Wonder if it'd be better to use top margin instead to make this a little less brittle?